### DISCUSSION TITLE
Naming convention for const in webdriverio

### DISCUSSION THREAD

**Discussion by durgaprbehera** [webdriverio/webdriverio#8833] [Technical Discussion]
As per the naming convention, the constant should be written in UPPERCASE.
But why are we not following the same here?
[https://webdriver.io/docs/selectors/](url)

So do we follow [https://google.github.io/styleguide/jsguide.html](url) ?

I need to set standards for my company's framework.

![image](https://user-images.githubusercontent.com/10437720/189256708-2b73e5fe-252c-495d-8497-3367cc8d4ea3.png)

**Comment by christian-bromann** [comment]
> As per the naming convention, the constant should be written in UPPERCASE.

Which naming convention. There is no general rule that all `const` vars have to be in uppercase. That is all up to you to define this. Also WebdriverIOs coding style should not impact the one you set for your company.
